QUICKSTART.md

```diff
@@ -0,0 +1,279 @@
+kube-system                       kube-apiserver-florin-control-plane                1/1     Running   0          58m
+kube-system                       kube-controller-manager-florin-control-plane       1/1     Running   0          58m
+kube-system                       kube-proxy-jmxsg                                   1/1     Running   0          58m
+kube-system                       kube-scheduler-florin-control-plane                1/1     Running   0          58m
+local-path-storage                local-path-provisioner-684f458cdd-kw2xz            1/1     Running   0          58m
+
+``` 
+
+Now, let's onboard the `guilder` cluster:
+
+```shell
+kubectl kcp-edge prep-for-cluster --imw root:example-imw guilder env=prod
+```
+
+Apply the created edge syncer manifest:
+
+```shell
+KUBECONFIG=$guilder_kubeconfig kubectl apply -f guilder-syncer.yaml
+```
+
+
+### e. Create and deploy the Apache Server workload into florin and guilder clusters
+
+Create the `EdgePlacement` object for your workload. Its “where predicate” (the locationSelectors array) has one label selector that matches the Location objects (`florin` and `guilder`) created earlier, thus directing the workload to both edge clusters.
+
+In the `wmw-1` workspace create the following `EdgePlacement` object: 
+  
+```shell
+kubectl ws root:my-org:example-wmw
+kubectl apply -f examples/common-placement.yaml
+```
+
+Deploy the Apache HTTP Server workload. Note the namespace label matches the label in the namespaceSelector for the EdgePlacement (`edge-placement-c`) object created above. 
+
+```shell
+kubectl apply -f examples/common-workload.yaml
+```
+
+Now, let's check that the deployment was created in the `florin` edge cluster:
+
+```shell
+KUBECONFIG=$florin_kubeconfig kubectl get deployments -A
+```
+
+which should yield something like:
+
+```console
+NAMESPACE     NAME      READY   UP-TO-DATE   AVAILABLE   AGE
+commonstuff   commond   0/0     0            0           6m44s
+```
+
+Also, let's check that the deployment was created in the `guilder` edge cluster:
+
+```shell
+KUBECONFIG=$guilder_kubeconfig kubectl get deployments -A
+```
+
+which should yield something like:
+
+```console
+NAMESPACE     NAME      READY   UP-TO-DATE   AVAILABLE   AGE
+commonstuff   commond   0/0     0            0           6m44s
+```
+
+Lastly, let's check that the workload is working in both clusters:
+
+For `florin`:
+
+```console
+$ curl http://localhost:8081
+<!DOCTYPE html>
+<html>
+  <body>
+    This is a common web site.
+  </body>
+</html>
+```
+
+For `guilder`:
+
+```console
+$ curl http://localhost:8082
+<!DOCTYPE html>
+<html>
+  <body>
+    This is a common web site.
+  </body>
+</html>
+```
+
+Congratulations, you’ve just deployed a workload to two edge clusters using kubestellar! To learn more about kubestellar please visit our [User Guide](<place-holder>)
+
+## 3. Cleanup the environment
```

Comment:
(1) I do not like the word "cleanup" here, it suggests that what has been done is somehow "dirty".  I think words like "removal", "teardown", and "undoing" are better because they carry no value connotations.

(2) The current text describes both commands as uninstalling kubestellar.  I think that is neither a good design nor an accurate description.

(3) A good design has teardown in the reverse order of the construction.  Thus it would go as follows.

> To remove the example usage, delete the IMW and WMW and kind clusters.  (Show the commands here).

> Stop and uninstall KubeStellar and kcp with the following commands (show them).  Note that start and stop are bundled with install and uninstall (respectively) in this QuickStart only for the sake of brevity.  Normally they are independent steps.  (Reference doc of those steps.)

Reply:
Fixed. I am proposing the following command flags: 
    1. `kubestellar remove`: to remove the example usage, delete the IMW and WMW and kind clusters
    2. `kubestellar stop`: to stop and uninstall KubeStellar and kcp